src/main/java/org/broadinstitute/hellbender/tools/exome/NormalizeSomaticReadCounts.java

```diff
@@ -119,7 +116,7 @@ protected Object doWork() {
                         "(" + HDF5PoNCreator.CURRENT_PON_VERSION + ").");
             }
 
-            final TargetCollection<? extends BEDFeature> exonCollection = readTargetCollection(targetFile);
+            final TargetCollection<Target> exonCollection = readTargetCollection(targetFile);
```

Comment:
Can we rename from ``exonCollection`` to something more appropriate, like ``targetCollection``?